src/textual/drivers/win32.py

```diff
@@ -162,6 +163,8 @@ def enable_application_mode() -> Callable[[], None]:
         A callable that will restore terminal to previous state.
     """
 
+    assert sys.__stdin__ is not None
```

Comment:
Let's add a message to these asserts. It is possible for `sys.__stdin__` to be `None`, although unlikely for Textual apps.